etc/bankdefs/hipo4/mc.json

```diff
@@ -100,5 +100,31 @@
             {"name":"procID",   "type":"I", "info":"Process that created the FP. see gemc.jlab.org"},
             {"name":"hitn",     "type":"I", "info":"Hit number"}
         ]
+    },
+    {
+        "name": "MC::IsParticleMatched",
+        "group": 40,
+        "item" : 5,
+        "info": "MC Particle - Rec Track matching.",
+        "entries": [
+            {"name":"mcTindex",      "type":"S",  "info":"MC Particle index, as in tid"},
+	    {"name":"pindex",	     "type":"S",  "info":"REC particle index"},
+	    {"name":"MCLayersTrk",   "type":"L",  "info":"layers of tracker (mostly)  detectors that a given MCParticle hit. Each layer corresponds to a specific bit in this variable"},
+	    {"name":"MCLayersNeut",  "type":"L",  "info":"layers of detectors (non tracker) that a givem MCParticle hit. Each layer corresponds to a specific bit in this variable"},
+	    {"name":"RecLayersTrk",  "type":"L",  "info":"layers of tracker detectors (mostly) that, the matched RecParticle picked hits from"},
+	    {"name":"RecLayersNeut", "type":"L",  "info":"layers of detectors (mostly) that, the matched RecParticle picked hits from"}
+        ]
+    },
+    {
+        "name": "MC::MatchedClusters",
```

Comment:
What additional information does thins bank provide with respect to the previous?

Reply:
You are right, this was used by Freancesco, and I believe it was just for debugging, but this bank is not filled anymore.
I the new commit it is removed.